tests/api/package/subpackage/api_4.py

```diff
@@ -0,0 +1,10 @@
+from http import HTTPStatus
+
+import winter
+
+
+@winter.route('api_4/')
+class API4:
```

Comment:
Probably it would be better to replace it with a valid code to avoid possible confusion. For example something without routes and api mentions.